.github/workflows/main.yml

```diff
@@ -0,0 +1,17 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: akhileshns/heroku-deploy@v3.12.12
+        with:
+          heroku_api_key: ${{secrets.HEROKU_API_KEY}}
+          heroku_app_name: "community-exchange"
+          heroku_email: "suzsalerno2@gmail.com"
```

Comment:
LGTM! only thing you might want to secret your email address. So you don't get spammed :)

Reply:
good catch! thanks - updated